httprouter/jsonrpcapi/encoding.go

```diff
@@ -0,0 +1,42 @@
+package jsonrpcapi
+
+import (
+	"encoding/hex"
+	"fmt"
+)
+
+// HexBytes is a []byte which encodes as hexadecimal in json, as opposed to the
+// base64 default.
+type HexBytes []byte
```

Comment:
do we not already have this somewhere?

Reply:
Yes, but import loops are annoying... Its just few lines that in some case is worth repeat it IMHO

Reply:
Ah. What about just moving the code to jsonrpcapi, and having the other packages pull HexBytes from here?

Another option is to move HexBytes to the types package.

Reply:
done